contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2083,7 +2083,7 @@ SELECT out_object_id as object_id
   , 1::sys.bit AS uses_database_collation
   , 0::sys.bit AS is_persisted
 FROM sys.columns_internal() sc
-INNER JOIN pg_attribute a ON sc.out_name = a.attname COLLATE sys.database_default AND sc.out_column_id = a.attnum
+INNER JOIN pg_attribute a ON sc.out_object_id = a.attrelid AND sc.out_name = a.attname COLLATE sys.database_default AND sc.out_column_id = a.attnum
```

Comment:
Otherwise we get invalid duplicates for two tables sharing the same column name, number.
Also affects scripting.

Reply:
Thought: Do we really need `sc.out_name = a.attname COLLATE sys.database_default` check?

Reply:
Yes, otherwise I was getting incorrect results
```
create table t1 (id INT, id1 as id*2)
create table t2 (id INT, id1 as id*2)

select name, OBJECT_NAME(object_id) from sys.computed_columns
go
name         object_name                                                                                                                     
id1             t1                                                                                                                              
id1             t2                                                                                                                              
id1             t1                                                                                                                              
id1             t2 
```

Reply:
oh, you meant attname check, i misread your comment.
No, we don't need that. I'll remove that.